apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -0,0 +1,89 @@
+/*global ServiceMixin */
+'use strict';
+
+importScripts('../service_mixin.js');
```

Comment:
Just theoretical question: should we use absolute or relative paths in ```importScripts```? I see here path is relative, but in mixin path is absolute.